services/app/src/components/views/Search.test.tsx

```diff
@@ -155,6 +169,18 @@ describe('Search', () => {
       expect(wrapper.text().toLowerCase()).not.toContain('last played');
     });
 
+    it('does not show book icon if it does not exist', () => {
+      const quest = FEATURED_QUESTS.filter((el) => el.title === 'Learning to Adventure')[0];
+      const {props, wrapper} = setup(quest.requirespenpaper, {requirespenpaper: false});
+      expect(wrapper.text().toLowerCase()).not.toContain('book');
+    });
+/*please review do I need wrapper.html?  this fails
+    it('shows a book icon if it exists', () => {
+      const quest = FEATURED_QUESTS.filter((el) => el.title === 'Learning to Adventure')[0];
+      const {props, wrapper} = setup(quest.requirespenpaper, {requirespenpaper: true});
+      expect(wrapper.text().toLowerCase()).toContain('book');
```

Comment:
Yep, since `book` is referencing the icon name, that must be rendered in the html / wouldn't show up in the text :)

Reply:
okay I will admit I don't know how to extend setup to include this - I thought it was covered with 'quest' as the third parameter, but this just doesn't pass.

Reply:
This code doesn't work.  Do I have to extend setup?  I figured since quest was included as SearchDetailsProp (ref to quest and therefore to requirespenpaper) I could reference it as 
quest.requirespenpaper: true, but that didn't work either.

setup(quest.requirespenpaper, {requirespenpaper: true});

Reply:
Can you describe how it doesn't work, ie what the error message is?

Reply:
okay that made sense!  The only problem now is that the expected value is not in the html (nor text) returned and the test still fails.  Here's what the failure looks like:

HeadlessChrome 0.0.0 (Linux 0.0.0) Search Details shows a book icon if it exists FAILED
	Expected '<div class="title_container"><button tabindex="0" class="MuiButtonBase-root-7 MuiIconButton-root-1" type="button" id="titlebarReturnButton"><span class="MuiIconButton-label-6"><svg class="MuiSvgIcon-root-10" focusable="false" viewBox="0 0 24 24" aria-hidden="true"><g><path d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"/></g></svg></span><span class="MuiTouchRipple-root-16"></span></button><span class="menu"><button tabindex="0" class="MuiButtonBase-root-7 MuiIconButton-root-1" type="button" aria-haspopup="true"><span class="MuiIconButton-label-6"><svg class="MuiSvgIcon-root-10" focusable="false" viewBox="0 0 24 24" aria-hidden="true"><g><path d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"/></g></svg></span><span class="MuiTouchRipple-root-16"></span></button></span><div class="title">Quest Details</div></div><div class="article"><div class="scrollbox"><div class="scrollbox_top ... to contain 'book'.
	    at <Jasmine>
	    at UserContext.<anonymous> (src/components/views/Search.test.tsx:201555:36)

Reply:
Can you push your code changes so that I can see what's going on?

Reply:
pushed, thanks.